packages/noports_core/lib/src/common/features.dart

```diff
@@ -0,0 +1,8 @@
+/// Features which can be supported by the NoPorts Daemon
+enum DaemonFeatures {
+  /// authenticate when connecting to the Socket Rendezvous (sr)
+  srAuth,
+
+  /// End-to-end encrypt traffic sent via the SocketRendezvous (sr)
+  srE2ee,
+}
```

Comment:
Should we add share public key to this list of features?

Reply:
Good idea!